src/components/Select/components/Listbox/Listbox.scss

```diff
@@ -0,0 +1,154 @@
+@import '../../../../../styles/mixins';
+@import '../../../variables';
+
+$block: '.#{$ns}select-listbox';
+$popupBlock: '.#{$ns}popup';
+$s-height: '28px';
+$s-hor-padding: '8px';
```

Comment:
Just `$s-padding` maybe? There is no other paddings.

Reply:
It using like this:
```scss
#{$block}_size_s & {
    height: #{$s-height};
    padding: 8px #{$s-hor-padding} 4px #{$s-hor-padding};
}
```
I think that variable must show that padding exactly horizontal